spatialmath/box.go

```diff
@@ -142,7 +142,7 @@ func (b *box) ToProtobuf() *commonpb.Geometry {
 }
 
 // CollidesWith checks if the given box collides with the given geometry and returns true if it does.
-func (b *box) CollidesWith(g Geometry) (bool, error) {
+func (b *box) CollidesWith(g Geometry, buffer float64) (bool, error) {
```

Comment:
Doesn't look like this is actually being used

Reply:
`checkCollision()` does